### ISSUE TITLE
Include notebooks into handbook?

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandi-docs#142]
Motivated by seeing how notebooks are included in https://nasa-impact.github.io/veda-docs/notebooks/quickstarts/open-and-plot.html manual .  That should improve their content indexing/findability by google.

WDYT?

**Comment by satra** [comment]
@yarikoptic - this uses the quarto framework for publishing. we could use that or jupyterbook, but it's not just including notebooks.

**Comment by yarikoptic** [comment]
Yes, we will need technical support for that one way or another: we use `mkdocs` so google lead me to https://github.com/danielfrg/mkdocs-jupyter , seems nice, could even (optionally) (re)execute notebooks, but I think we shouldn't even try, but eventually might want to get there.
